news/2024/2024-11-20-osu-world-cup-2024-quarterfinals-recap.md

```diff
@@ -0,0 +1,154 @@
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/cld2spAZQ-M?start=792&end=947" frameborder="0" allowfullscreen></iframe>
+</div>
+
+The best match of the Quarterfinals may have been one you didn't even know about or watch. With three matches all happening at once, ::{ flag=PH }:: Philippines (seed #9) vs. ::{ flag=UA }:: Ukraine (seed #22), drew the short straw of not being streamed on the official Twitch channels. As it happened, this unofficial stream would include the only tiebreaker match of the weekend
+
+Both teams had similar strengths yet different weaknesses coming into this match, resulting in a back-and-forth slugfest. The ::{ flag=PH }:: Philippines opened with [DT3](https://osu.ppy.sh/beatmapsets/2072797#osu/4419768) as their first pick, but ::{ flag=UA }:: Ukraine took it as a break point to start the match, consolidating with their own [DT1](https://osu.ppy.sh/beatmapsets/209170#osu/492285) first pick. The ::{ flag=PH }:: Philippines tried switching gears to [NM4](https://osu.ppy.sh/beatmapsets/2278186#osu/4855557), but with three scores above 700,000, ::{ flag=UA }:: Ukraine ended up with 2 break points for a 3-0 lead.
+
+Momentum quickly shifted with ::{ flag=UA }:: Ukraine's next pick, the breakcore-style speed [FM2](https://osu.ppy.sh/beatmapsets/2278184#osu/4855553). The ::{ flag=PH }:: Philippines took this map, won their own next [HD2](https://osu.ppy.sh/beatmapsets/2278079#osu/4855234) pick with a whopping 905,684 coming from ::{ flag=PH }:: [Rammu](https://osu.ppy.sh/users/10652837), and stole ::{ flag=UA }:: Ukraine's HR2 pick afterwards with a carry performance of 839,244 from ::{ flag=PH }:: [zonelouise](https://osu.ppy.sh/users/1492995), tying the match 3-3.
+
+Another two very close break points followed on [HD3](https://osu.ppy.sh/beatmapsets/1381824#osu/3004925) and [NM2](https://osu.ppy.sh/beatmapsets/2278182#osu/4855537) with very close 200,000 score differences on both maps, bringing the match to 4-4. Both teams had saved a good last pick for this moment, with the ::{ flag=PH }:: Philippines picking [NM5](https://osu.ppy.sh/beatmapsets/2020320#osu/4207467), the AR 5 reading pick, and ::{ flag=UA }:: Ukraine picking [HR3](https://osu.ppy.sh/beatmapsets/908811#osu/1896581), the flow alt tech hybrid map. With the teams tied 5-5, the only tiebreaker of the Quarterfinals weekend was secured.
+
+Both these teams had something to prove going into this tiebreaker — the ::{ flag=PH }:: Philippines and ::{ flag=UA }:: Ukraine both lost at least one key team member from last year, with [enri](https://osu.ppy.sh/users/8640970) and [hyeok2044](https://osu.ppy.sh/users/8472976) leaving the ::{ flag=PH }:: Philippines and [dench](https://osu.ppy.sh/users/10073635) leaving ::{ flag=UA }:: Ukraine. Repeating or going beyond their final placement last year would mean that these teams can hold their own year to year regardless of roster changes.
```

Comment:
were enri's and hyeok's flags excluded for a reason (flag change) or?

Reply:
https://discord.com/channels/188630481301012481/1293209725034303529/1308941695106416690